<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>translator_xyz</class>
 <widget class="QWidget" name="translator_xyz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Select the Region of Interest to translate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_meaCoor"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>XY-reference location</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="1" column="2">
                  <widget class="QRadioButton" name="btn_xy_centreright">
                   <property name="text">
                    <string>Centre right</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QRadioButton" name="btn_xy_centrecentre">
                   <property name="text">
                    <string>Centre centre</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QRadioButton" name="btn_xy_topright">
                   <property name="text">
                    <string>Top right</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QRadioButton" name="btn_xy_bottomright">
                   <property name="text">
                    <string>Bottom right</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QRadioButton" name="btn_xy_centreleft">
                   <property name="text">
                    <string>Centre left</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="btn_xy_topleft">
                   <property name="text">
                    <string>Top left</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QRadioButton" name="btn_xy_bottomleft">
                   <property name="text">
                    <string>Bottom left</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QRadioButton" name="btn_xy_bottomcentre">
                   <property name="text">
                    <string>Bottom centre</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QRadioButton" name="btn_xy_topcentre">
                   <property name="text">
                    <string>Top centre</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Z-reference location</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="2" column="0">
                  <widget class="QRadioButton" name="btn_z_bottom">
                   <property name="text">
                    <string>Bottom</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QRadioButton" name="btn_z_centre">
                   <property name="text">
                    <string>Centre</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="btn_z_top">
                   <property name="text">
                    <string>Top</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btn_goto_refLoc">
            <property name="text">
             <string>Move stage to the reference location</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Previous coordinate (µm):</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="5">
             <widget class="QPushButton" name="btn_commit">
              <property name="text">
               <string>Perform XYZ translation</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="spin_coorYUm">
              <property name="minimum">
               <double>-1000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QPushButton" name="btn_storeCoor">
              <property name="text">
               <string>Insert current coordinate</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbl_prevCoor">
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>New coordinate (µm):</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spin_coorXUm">
              <property name="minimum">
               <double>-1000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QDoubleSpinBox" name="spin_coorZUm">
              <property name="minimum">
               <double>-1000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="chk_autoSelect">
            <property name="text">
             <string>Auto-select the last modified (translated) ROI</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chk_autoMove">
            <property name="text">
             <string>Auto-move the stage using the last modified translation</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chk_automove_xyonly">
            <property name="text">
             <string>Auto-move only in the XY-direction (exclude Z)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_instructions">
       <property name="text">
        <string>Show instructions</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_meaCoor</tabstop>
  <tabstop>btn_xy_topleft</tabstop>
  <tabstop>btn_xy_topcentre</tabstop>
  <tabstop>btn_xy_topright</tabstop>
  <tabstop>btn_xy_centreleft</tabstop>
  <tabstop>btn_xy_centrecentre</tabstop>
  <tabstop>btn_xy_centreright</tabstop>
  <tabstop>btn_xy_bottomleft</tabstop>
  <tabstop>btn_xy_bottomcentre</tabstop>
  <tabstop>btn_xy_bottomright</tabstop>
  <tabstop>btn_z_top</tabstop>
  <tabstop>btn_z_centre</tabstop>
  <tabstop>btn_z_bottom</tabstop>
  <tabstop>btn_goto_refLoc</tabstop>
  <tabstop>spin_coorXUm</tabstop>
  <tabstop>spin_coorYUm</tabstop>
  <tabstop>spin_coorZUm</tabstop>
  <tabstop>btn_storeCoor</tabstop>
  <tabstop>btn_commit</tabstop>
  <tabstop>chk_autoSelect</tabstop>
  <tabstop>chk_autoMove</tabstop>
  <tabstop>btn_instructions</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
